tests/common/devices.py

```diff
@@ -350,6 +350,24 @@ def get_image_info(self):
     def get_asic_type(self):
         return self.facts["asic_type"]
 
+    def shutdown(self, portname):
+        """
+            Shutdown interface (parity function as EosHost)
```

Comment:
I think I get what you're describing here but "parity function as EosHost" is not very clear to me.

Reply:
EosHost has shutdown and no_shutdown method to set interface state. To have function with same name would help code unity.

How do you suggest to word this comment?